### PR TITLE
test pull request

### PR DESCRIPTION
**Pull request by koungeiei**
comment naja

### CODE REVIEW

**Comment by koungeiei** [comment]
comment 2

File: README.md [hunk 1]
```diff
@@ -24,18 +24,4 @@ this project not following [golang-project-layout](https://github.com/golang-sta
 
 `make docker`
 
-## Folder Explain
-
-`main.go` main program
-
-`src/connection` for 3rd party connection
-
-`src/controller` controller in `MVC` model
```
Comment (koungkub) [inline README.md:33]: eiei

File: README.md [hunk 2]
```diff
@@ -24,18 +24,4 @@ this project not following [golang-project-layout](https://github.com/golang-sta
...
-
-`src/middleware` custom middleware
-
-`src/route` setup middleware and routing
```
Comment (koungkub) [inline README.md:37]: helloworld

**Review by koungkub — COMMENTED** [review]
commendddddddd

**Pull request merged**